File: Usuario/backend/src/main/java/com/signspeak/backend/dto/FiltrosDiccionarioRequest.java
package com.signspeak.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FiltrosDiccionarioRequest {
    private String nombre;  // ✅ Cambiado de "palabra" a "nombre"
    private Long idCategoria;
    private String dificultad;  // "FACIL", "MEDIO", "DIFICIL"
    private String ordenar; // "popularidad", "alfabetico", "reciente"
    private Integer pagina;
    private Integer tamanoPagina;
}

File: Usuario/mobile/app/src/main/java/com/signspeak/data/remote/dto/diccionario/FiltrosDiccionarioRequest.kt
package com.signspeak.data.remote.dto.diccionario

data class FiltrosDiccionarioRequest(
    val nombre: String? = null,
    val idCategoria: Long? = null,
    val dificultad: String? = null, // "FACIL", "MEDIO", "DIFICIL"
    val ordenar: String? = null, // "popularidad", "alfabetico", "reciente"
    val pagina: Int = 0,
    val tamanoPagina: Int = 20
)
